.about.yml

```diff
@@ -0,0 +1,113 @@
+---
+# .about.yml project metadata
+#
+# Short name that acts as the project identifier (required)
+name: project-dashboard
+
+# Full proper name of the project (required)
+full_name: 'Project Dashboard'
```

Comment:
What do you think of calling it "Project Status Dashboard", to avoid confusion with the [18F Dashboard](https://18f.gsa.gov/dashboard/)?